src/test/java/nextstep/subway/station/StationAcceptanceTest.java

```diff
@@ -111,5 +100,39 @@ void getStations() {
     @DisplayName("지하철역을 제거한다.")
     @Test
     void deleteStation() {
+        // given
+        ValidatableResponse validatableResponse = 지하철역_생성("강남역");
+        long createStationId = validatableResponse.extract().jsonPath().getLong("id");
+
+        // when
+        ExtractableResponse<Response> deleteResponse =
+            RestAssured.given().log().all()
+                .pathParam("id", createStationId)
+                .when().delete("/stations/{id}")
+                .then().log().all()
+                .extract();
+
+        // then
+        assertThat(deleteResponse.statusCode()).isEqualTo(HttpStatus.NO_CONTENT.value());
+
+        // then
+        List<String> selectResponse = 지하철역_전체_조회().jsonPath().getList("$");
+        assertThat(selectResponse).hasSize(0);
```

Comment:
```suggestion
        assertThat(selectResponse).isEmpty();
```
Collection의 크기를 0과 비교하는 경우엔 이 메소드가 조금 더 눈에 잘 띄는 효과가 있어요.